src/java.desktop/windows/native/libawt/windows/awt_PrintJob.cpp

```diff
@@ -3925,9 +3925,13 @@ static void throwPrinterException(JNIEnv *env, DWORD err) {
                   sizeof(t_errStr),
                   NULL );
 
-    WideCharToMultiByte(CP_UTF8, 0, t_errStr, -1,
+    int nb = WideCharToMultiByte(CP_UTF8, 0, t_errStr, -1,
                         errStr, sizeof(errStr), NULL, NULL);
-    JNU_ThrowByName(env, PRINTEREXCEPTION_STR, errStr);
+    if (nb > 0) {
+        JNU_ThrowByName(env, PRINTEREXCEPTION_STR, errStr);
+    } else {
+        JNU_ThrowByName(env, PRINTEREXCEPTION_STR, "Secondary error while OS message extraction");
```

Comment:
better: secondary error during OS message extraction

Reply:
I borrowed it from here  (other error handling of WideCharToMultiByte) 
```

jdk/src/java.base/windows/native/libjava/ProcessImpl_md.c 

102          const char *errorMessage = "Secondary error while OS message extraction";
```

should I change both locations ?